index.html

```diff
@@ -964,7 +975,7 @@
             <dt>-to 00:55:00</dt><dd>sets out point at 00:55:00</dd>
             <dt>-c copy</dt><dd>use stream copy mode (no re-encoding)<br>
             <dt>-map 0</dt><dd>Tells ffmpeg to map all streams of the input to the output.</dd>
-            <i>Note:</i> watch out when using <code>-ss</code> with <code>-c copy</code> if the source is encoded with an interframe codec (e.g., H.264). Since ffmpeg must split on i-frames, it will seek to the nearest i-frame to begin the stream copy.</dd>
+            <b>Note</b>: watch out when using <code>-ss</code> with <code>-c copy</code> if the source is encoded with an interframe codec (e.g., H.264). Since ffmpeg must split on i-frames, it will seek to the nearest i-frame to begin the stream copy.</dd>
```

Comment:
I would let also the : as bold.

Reply:
I have an aversion to bolding colons in such positions, I don't know why.

Reply:
I don’t know the rules in English. In the languages I know the interpunction signs immediately following a word are set in the same type.

Reply:
It feels wrong to me, but the Chicago Manual of Style indeed indicates that the colons should be bolded. I'll change them.